src/components/edit-asset-address-form/editable-address/editable-address.tsx

```diff
@@ -85,7 +121,7 @@ export const EditableAddress = ({
         };
         setCurrentAssetAddress(newAssetAddress);
         setShowSuccess(true);
-        setSubmitEditEnabled(false);
+        setAddressEditSuccessful(true);
```

Comment:
What if they want to make another edit?

Reply:
They have to re-visit the Edit page
I've run Sean through this behaviour and he's happy with it.
I think if the users need the ability to submit multiple edits ("at once") we can change this relatively easily in the future